DBImport/InsertData.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+"""Insert Data component.
+
+This module gets lists of Course and Department objects from ReadCourseData.py
+and insert those data into desired databse
+"""
+
+import logging
+import os
+from ReadCourseData import *
+from configparser import ConfigParser
+from pymongo import MongoClient
+from dotenv import load_dotenv
+from pathlib import Path
+
+
+def get_db():
+    """Get MongoDB username and password from .env file and returns desired databse.
+
+    Args:
+        'Mongo_User'(from '.env'): the user name used to access the mongoDB Atlas
+        'Mongo_Password'(from '.env'): the password used to access the mongoDB Atlas
+        'Mongo_DBName'(from '.env'): the desired database name
+        'Mongo_Postfix'(from '.env'): the postfix of the srv link from MongoDB
+    Raises:
+        pymongo.errors: possibly connection errors or conficuration errors
+    Returns:
+        The database object
+
+    """
+    env_path = Path('.') / '.env'
```

Comment:
um... we may create `.env` folder.
There should have a batter way to do this.
maybe import `config file` under this folder?